.github/workflows/release.yml

```diff
@@ -13,3 +13,17 @@ jobs:
       push: true
       version: ${{ github.event.release.tag_name }}
     secrets: inherit
+
+  deploy_release:
+    name: deploy to https://otel-demo.eu-west-1.aws.dash0-demo.com/
+    needs:
+      - build_and_push_images
+    runs-on: ubuntu-latest
+    # do not run multiple deployment jobs concurrently
+    concurrency: deploy
+    steps:
+      - name: deploy release
+        uses: .github/actions/deploy-development
```

Comment:
Typo: path should be "deploy-demo" or you rename the directory

Reply:
right!